### PR TITLE
vscode@1.96.2: Fix hash

### LINKED ISSUE #182: vscode@1.96.2: hash check failed
ERROR Hash check failed!
App:         business/vscode
URL:         https://vscode.download.prss.microsoft.com/dbazure/download/stable/fabdb6a30b49f79a7aba0f2ad9df9b399473380f/VSCode-win32-x64-1.96.2.zip
First bytes: 50 4B 03 04 14 00 00 00
Expected:    e9a4af2c79c7a7c1208b5275ab7cc3709135153882a026a0418964dc9b65f387
Actual:      c6c2f97e5cb25a8b576b345f6b8f2021cc168f1726ee370f29d1dbd136ffe9f8

### PR DESCRIPTION
**Pull request by BotServiceAutomation**
- Closes #182

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### vscode

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Review by alexchapar — APPROVED** [review]

**Pull request merged**
